lib/index.js

```diff
@@ -18,52 +20,69 @@ var Slack = module.exports = integration('Slack')
   .retries(1);
 
 /**
+ * Ensure.
+ */
+
+Slack.ensure(function(msg, settings) {
+  if (msg.type() === 'identify') {
+    if (settings.whiteListedTraits.length === 0) return this.reject('No white listed traits were set.');
+    if (!includesKeys(msg.traits(), settings.whiteListedTraits)) return this.reject('Only identify calls that include `traits` in settings will be sent.');
```

Comment:
`'Only identify calls that have match traits listed in the settings will be sent.'`